tests/codegen-units/partitioning/README.md

```diff
@@ -0,0 +1,14 @@
+# codegen-units/partitioning tests
+
+This test suite is designed to test that codegen unit partitioning works as intended.
+Note that it does not evaluate whether CGU partitioning is *good*, that is the job of the compiler benchmark suite.
```

Comment:
This sentence has a [comma splice](https://www.grammarly.com/blog/punctuation-capitalization/comma-splice/) which you don't need to fix but lots of people don't know about them so I thought I'd mention it for educational purposes :)  (I'd change the comma to a period to fix it.)